spring-batch-core-tests/src/test/java/org/springframework/batch/core/test/step/FaultTolerantStepIntegrationTests.java

```diff
@@ -0,0 +1,169 @@
+package org.springframework.batch.core.test.step;
```

Comment:
While not as big of a deal on tests, I recommend having your IDE generate the license at the top of any java file you create.

Reply:
ok, I've updated my IDE for that.